### PR TITLE
Remove circular imports

### PR DESCRIPTION
**Pull request by ibrahimsharaf**
calling downloader script from utils and utils from downloader caused import errors due to circular imports, so I moved the stack traces related functions to a new utils script.

### CODE REVIEW

File: crashsimilarity/downloader.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import os
-from datetime import timedelta
```
Comment (marco-c) [inline crashsimilarity/downloader.py]: Nit: Don't move this import

File: crashsimilarity/stacktrace_utils.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import json
```
Comment (marco-c) [inline crashsimilarity/stacktrace_utils.py]: Could you rename this file `stacktrace.py` directly?

Comment (ibrahimsharaf) [reply crashsimilarity/stacktrace_utils.py]: Fixed @marco-c

File: crashsimilarity/downloader.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 import logging
+from datetime import timedelta
```
Comment (marco-c) [inline crashsimilarity/downloader.py]: Could you move this back? I'd like to have clean history and so avoid unrelated changes

Comment (ibrahimsharaf) [reply crashsimilarity/downloader.py]: Done
